### ISSUE TITLE
Code issue in Scale Expand PVC TC

### ISSUE THREAD

**Issue by ramkiperiy** [red-hat-storage/ocs-ci#2760]
Scale PVC expand testcase failed due to code issue listed below

```
____________________________________________________________ TestPVCExpand.test_scale_pvc_expand[False-10Gi-20] _____________________________________________________________

self = <test_scale_pvc_expand.TestPVCExpand object at 0x7fe06350cca0>, resize_pvc = <ocs_ci.ocs.scale_lib.FioPodScale object at 0x7fdf89d28040>, start_io = False
pvc_size = '10Gi', pvc_new_size = 20

    def test_scale_pvc_expand(
        self, resize_pvc, start_io, pvc_size, pvc_new_size
    ):
        """
        Test case to scale pvc size expansion
        with and without IO running on the pods
        """
        # Create pvcs and scale pods
        logging.info('Create pvcs and scale pods')
>       resize_pvc.create_scale_pods(
            scale_count=1500, pods_per_iter=10,
            io_runtime=3600, start_io=start_io, pvc_size=pvc_size
        )

tests/e2e/scale/test_scale_pvc_expand.py:57:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
ocs_ci/ocs/scale_lib.py:223: in create_scale_pods
    self.pod_obj, self.pvc_obj = self.create_multi_pvc_pod(
ocs_ci/ocs/scale_lib.py:102: in create_multi_pvc_pod
    fio_rate = get_rate_based_on_cls_iops()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
ocs_ci/ocs/scale_lib.py:223: in create_scale_pods
    self.pod_obj, self.pvc_obj = self.create_multi_pvc_pod(
ocs_ci/ocs/scale_lib.py:102: in create_multi_pvc_pod
    fio_rate = get_rate_based_on_cls_iops()
ocs_ci/ocs/scale_lib.py:505: in get_rate_based_on_cls_iops
    iops = cls_obj.get_iops_percentage(osd_size=osd_size)
ocs_ci/ocs/cluster.py:566: in get_iops_percentage
    iops_in_cluster = self.get_ceph_cluster_iops()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <ocs_ci.ocs.cluster.CephCluster object at 0x7fe2e214f970>

    def get_ceph_cluster_iops(self):
        """
        The function gets the IOPS from the ocs cluster
    
        Returns:
            Total IOPS in the cluster
    
        """
    
        ceph_pod = pod.get_ceph_tools_pod()
        ceph_status = ceph_pod.exec_ceph_cmd(ceph_cmd="ceph status")
>       read_ops = ceph_status['pgmap']['read_op_per_sec']
E       KeyError: 'read_op_per_sec'

ocs_ci/ocs/cluster.py:546: KeyError
```

**Comment by ramkiperiy** [comment]
@tiffanyn108 Please check this.

**Comment by wusui** [comment]
Has this been resolved yet?  I am running into the same problem running tests/e2e/performance/test_small_file_workload.py

**Comment by tiffanyn108** [comment]
@wusui The test case failed due to creating 1500 PVCs and pods which is related to memory issue.  
It shouldn't be seen when test is executed from a slave machine with 32GB of memory.

**Comment by tiffanyn108** [comment]
This PR will be used to track the code change with new scale_lib to avoid memory issue.

**Comment by tiffanyn108** [comment]
fixed in #4111
